sermon-mount.txt

```diff
@@ -1,215 +1,215 @@
+Nor by the earth, for it is his footstool;
 
-Neither shalt thou swear by thy head, because thou canst not make one hair white or black.
+Neither shalt thou swear by thy head, because thou canst not make one hair black or white;
 
-But let your communication be, Yea, yea; Nay, nay: for whatsoever is more than these cometh of evil.
+But let your communication be Yea, yea; Nay, nay; for whatsoever cometh of more than these is evil.
 
-Ye have heard that it hath been said, An eye for an eye, and a tooth for a tooth:
+And behold, it is written, an eye for an eye, and a tooth for a tooth;
 
-But I say unto you, That ye resist not evil: but whosoever shall smite thee on thy right cheek, turn to him the other also.
+But I say unto you, that ye shall not resist evil, but whosoever shall smite thee on thy right cheek, turn to him the other also;
 
-And if any man will sue thee at the law, and take away thy coat, let him have thy cloak also.
+And if any man will sue thee at the law and take away thy coat, let him have thy cloak also;
 
 And whosoever shall compel thee to go a mile, go with him twain.
 
-Give to him that asketh thee, and from him that would borrow of thee turn not thou away.
+Give to him that asketh thee, and from him that would borrow of thee turn thou not away.
 
-Ye have heard that it hath been said, Thou shalt love thy neighbour, and hate thine enemy.
+And behold it is written also, that thou shalt love thy neighbor and hate thine enemy;
 
-But I say unto you, Love your enemies, bless them that curse you, do good to them that hate you, and pray for them which despitefully use you, and persecute you;
+But behold I say unto you, love your enemies, bless them that curse you, do good to them that hate you, and pray for them who despitefully use you and persecute you;
 
-That ye may be the children of your Father which is in heaven: for he maketh his sun to rise on the evil and on the good, and sendeth rain on the just and on the unjust.
+That ye may be the children of your Father who is in heaven; for he maketh his sun to rise on the evil and on the good.
 
-For if ye love them which love you, what reward have ye? do not even the publicans the same?
+Therefore those things which were of old time, which were under the law, in me are all fulfilled.
 
-And if ye salute your brethren only, what do ye more than others? do not even the publicans so?
+Old things are done away, and all things have become new.
 
-Be ye therefore perfect, even as your Father which is in heaven is perfect. Take heed that ye do not your alms before men, to be seen of them: otherwise ye have no reward of your Father which is in heaven.
+Therefore I would that ye should be perfect even as I, or your Father who is in heaven is perfect. Verily, verily, I say that I would that ye should do alms unto the poor; but take heed that ye do not your alms before men to be seen of them; otherwise ye have no reward of your Father who is in heaven.
```

Comment:
at this point, in the Book of Mormon, Jesus was already perfected, i.e. had been resurrected, so he identified himself as being perfect just as His Father.